firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -28,6 +28,15 @@ def test_focus_and_clear(self):
         locationbar.clear()
         self.assertEqual(locationbar.value, '')
 
+    def test_gobutton(self):
```

Comment:
For unit level tests of properties we usually only check some of its attributes. Mainly the `localName`. See https://github.com/rbillings/firefox-ui-tests/blob/gobutton/firefox_puppeteer/tests/test_toolbars.py#L48 as reference.

So the test as written here is way more complex and should really end-up in firefox_ui_tests/functional.